Formula/coredns.rb

```diff
@@ -0,0 +1,84 @@
+class Coredns < Formula
+  desc "DNS server that chains plugins"
+  homepage "https://coredns.io"
+  url "https://github.com/coredns/coredns.git",
+    :tag => "v1.0.3",
+    :revision => "dc6c40740f882d6edc38bd1714a597ff65bf064e"
+  head "https://github.com/coredns/coredns.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOOS"] = "darwin"
+    ENV["GOARCH"] = MacOS.prefer_64_bit? ? "amd64" : "386"
+
+    (buildpath/"src/github.com/coredns/coredns").install buildpath.children
+
+    cd "src/github.com/coredns/coredns" do
+      system "make", "coredns", "CHECKS=godeps"
```

Comment:
I'm mildly confused what upstream is doing with `CHECKS=godeps` given they've implemented the vendor system. Any idea why they've left out certain deps?

Reply:
They had various dependency issues using vendoring. 
Here are the related issues.
- coredns/coredns#696
- coredns/coredns#792
- coredns/coredns#793
- coredns/coredns#794